### PR TITLE
permissions.yml lucy

### PR DESCRIPTION
**Pull request by adambash**

### CODE REVIEW

**Comment by adambash** [comment]
from the tv show Elfen Lied       requested by Suannaii

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -2973,6 +2973,33 @@ groups:
...
+    - mortal
+    options:
+      prefix: '&fL&du&fc&dy'
+      suffix: '&d'
+    worlds:
+      survival:
+        permissions:
+        - ch.alias.kill
+        - ch.alias.vibrate
+        - ch.alias.stopvibrating
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: requires proof of vibrating

File: PermissionsEx/permissions.yml [hunk 2]
```diff
@@ -2973,6 +2973,33 @@ groups:
...
+  lucy:
+    inheritance:
+    - mortal
+    options:
+      prefix: '&fL&du&fc&dy'
+      suffix: '&d'
+    worlds:
+      survival:
+        permissions:
+        - ch.alias.kill
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: requires proof of killing via willpower

**Comment by malonnnn** [comment]
who is lucy?

**Comment by malonnnn** [comment]
lucy is ambiguous, please be more specific on which hero this is

**Pull request closed without merging**